pages/dkp/konvoy/2.1/choose-infrastructure/pre-provisioned/create-cluster/index.md

```diff
@@ -28,6 +28,39 @@ Depending on the cluster size, it will take a few minutes to be created. After t
 dkp get kubeconfig -c ${CLUSTER_NAME} > ${CLUSTER_NAME}.conf
 ```
 
+### Confirming `calico` installation
+
+Before exploring the new cluster further, you will want to confirm your `calico` installation is correct.
+In the default case, Calico will automatically detect the IP to use for each node using the `first-found` [method][calico-method]. This may not be appropriate in our particular nodes and you will need to modify Calico's configuration to use a different method.
```

Comment:
```suggestion
By default, Calico automatically detects the IP to use for each node using the `first-found` [method][calico-method]. This may not be appropriate in your particular nodes, and you must modify Calico's configuration to use a different method.
```